lib/widgets/autocomplete.dart

```diff
@@ -208,6 +208,19 @@ class ComposeAutocomplete extends AutocompleteField<ComposeAutocompleteQuery, Co
         replacementString = '${userMention(user, silent: query.silent, users: store)} ';
       case WildcardMentionAutocompleteResult(:var wildcardOption):
         replacementString = '${wildcardMention(wildcardOption, store: store)} ';
+      case UserGroupMentionAutocompleteResult(:final id):
+        if (query is! MentionAutocompleteQuery) {
+          return; // Shrug; similar to `intent == null` case above.
+        }
+        final userGroup = store.getGroup(id);
+        if (userGroup == null) {
+          // Don't crash on theoretical race between async results-filtering
+          // and losing data for the group.
+          return;
```

Comment:
Hmm interesting. I guess if that is possible, it probably also applies to the user case above:
```dart
        final user = store.getUser(userId)!; // must exist because UserMentionAutocompleteResult
```